### ISSUE TITLE
Circle Looks more like Hexagon when s ized

### ISSUE THREAD

**Issue by aDioS786** [epezent/implot#366]
Hi
Not sure why circle looks more like hexagon, and also if the weight is increased it the boundaries looks like flower.

Circle Code:
	ImPlot::SetNextMarkerStyle(ImPlotMarker_Circle, radiusSize, FillColor, thickness, OutFillColor);
	ImPlot::PlotScatter(name, &xyCoordinate[0], &xyCoordinate[1], 1);

![image](https://user-images.githubusercontent.com/76258953/173258531-ef7484b8-ddc8-451c-b3bb-1827926ad6ac.png)

And this is how it looks like if thickness is increased:

![image](https://user-images.githubusercontent.com/76258953/173258555-044db646-8b74-4845-8616-2446bdc516ca.png)

**Comment by lesauxvi** [comment]
Because the circle is rasterized with 10 points. You can have a look at the method called `RenderMarkerCircle` in the file `implot_items.cpp`.

**Comment by epezent** [comment]
Correct. It's not expected for anyone to render markers this large, so circles only use 10 points to keep their cost low. If you need large circles I suggest custom rendering.

**Comment by aDioS786** [comment]
> Correct. It's not expected for anyone to render markers this large, so circles only use 10 points to keep their cost low. If you need large circles I suggest custom rendering.



> Because the circle is rasterized with 10 points. You can have a look at the method called `RenderMarkerCircle` in the file `implot_items.cpp`.

Many Thanks, also how can we print text on x,y coordinates, I am using ImPlot::GetPlotDrawList()->AddText(ImPlot::PlotToPixels( x,y)) but when i zoom-in the text doesn't scale automatically, whereas the implot draw functions automatically scale objects based on zoom level. Can't seem to find anything to draw text. Thanks

**Comment by epezent** [comment]
> Many Thanks, I'll try find where it's setting/using 10 points and maybe increase them a bit. Any plan on adding points as an additional parameter in future so we can use increased params if needed?

No, I don't plan on making points an argument to `PlotScatter` etc. since it does not apply to any other marker types. But, I do have plans to add a `PlotBubbles` (see #298). That plotter will probably need a smart way of determining the number of req'd points so that the bubbles are smooth at all sizes. Stay tuned.

**Comment by aDioS786** [comment]
That be a nice features looks great. Huge thanks to you for ur hard effort doing this project.

One more questions we have a legend on left, is there a way or any built in function we got, say I want to click on first item in legend, when clicked i want the graph or canvas area to focus on that legend/plotted item. Is that possible to pick coordinates and fo cus on that area on click? Thanks
